SpringProject/src/main/java/com/gradproject2019/conferences/service/ConferenceServiceImpl.java

```diff
@@ -60,4 +63,13 @@ private void checkConferenceExists(Long conferenceId) {
             throw new ConferenceNotFoundException();
         }
     }
+
+    @Override
+    public ConferenceResponseDto editConference(Long conferenceId, ConferencePatchRequestDto conferencePatchRequestDto){
+        checkConferenceExists(conferenceId);
+        Conference conference = conferencePatchRequestDto.from(conferenceId, conferencePatchRequestDto);
```

Comment:
You don't need to convert the conferencePatchRequestDto to a Conference here, you can do everything you need without doing that